packages/opds-web-client/src/utils/book.ts

```diff
@@ -0,0 +1,21 @@
+import { BookData, LinkData } from "./../interfaces";
+
+/**
+ *  A collection of utils for processing book data
+ */
+
+export function bookIsReserved(book: BookData) {
+  return book.availability?.status === "reserved";
+}
+
+export function bookIsReady(book: BookData) {
+  return book.availability?.status === "ready";
+}
+
+export function bookIsBorrowed(book: BookData) {
+  return book.fulfillmentLinks?.length ?? 0 > 0;
```

Comment:
Why is the second clause `0 > 0`? This means that if and only if `length` is null or undefined, then check that 0 is greater than 0?

Reply:
Oh shoot you're right I missed a parenthesis in there